lib/src/utils/navigation.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+
+class Navigation {
+  const Navigation._();
+
+  static Future<T?> push<T extends Object?>(
+      BuildContext context, Widget nextPage) async {
+    return await Navigator.of(context).push(MaterialPageRoute(
```

Comment:
Please use go_router because this app may need a deep link someday. It is not easy to implement deep link with Navigator 1.0.